runtime/centrifuge/src/migrations.rs

```diff
@@ -11,7 +11,17 @@
 // GNU General Public License for more details.
 use crate::{Runtime, Weight};
 
-pub type UpgradeCentrifuge1021 = (anemoy_pool::Migration, add_wrapped_usdc_variants::Migration);
+#[cfg(not(feature = "testnet-runtime"))]
+pub type UpgradeCentrifuge1022 = (
+	anemoy_pool::Migration,
+	add_wrapped_usdc_variants::Migration,
+	// Sets account codes for all precompiles
+	runtime_common::migrations::precompile_account_codes::Migration<crate::Runtime>,
+);
+
+#[cfg(feature = "testnet-runtime")]
+pub type UpgradeCentrifuge1022 =
+	(runtime_common::migrations::precompile_account_codes::Migration<crate::Runtime>);
```

Comment:
I'd love if we did not introduce a `testnet` runtime to Catalyst as the purpose of this chain is to be an exact testnet copy of the Centrifuge runtime such that we test exactly the WASM which we propose to Centrifuge chain. We can handle the migrations well enough with spec version checks.

So I would like drop this feature immediately after upgrading Centrifuge to 1022. Otherwise this will lead to diverging runtimes which is not the purpose of Catalyst.